### PR TITLE
Add mobile-event parquet schema.

### PR DESCRIPTION
**Pull request by justindarc**

### CODE REVIEW

**Comment by justindarc** [comment]
@fbertsch

File: templates/telemetry/mobile-event/mobile-event.1.parquetmr.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+message mobile_event {
+  required group metadata {
+    required binary documentId (UTF8);
+    required int64  Timestamp;
+    optional binary Date (UTF8);
+    optional binary geoCountry (UTF8);
```
Comment (fbertsch) [inline templates/telemetry/mobile-event/mobile-event.1.parquetmr.txt]: make this `required`

File: templates/telemetry/mobile-event/mobile-event.1.parquetmr.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    optional binary geoCity (UTF8);
```
Comment (fbertsch) [inline templates/telemetry/mobile-event/mobile-event.1.parquetmr.txt]: `required`

File: templates/telemetry/mobile-event/mobile-event.1.parquetmr.txt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    optional binary appBuildId (UTF8);
```
Comment (fbertsch) [inline templates/telemetry/mobile-event/mobile-event.1.parquetmr.txt]: `required`

File: templates/telemetry/mobile-event/mobile-event.1.parquetmr.txt [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    optional binary appName (UTF8);
```
Comment (fbertsch) [inline templates/telemetry/mobile-event/mobile-event.1.parquetmr.txt]: `required`

**Comment by justindarc** [comment]
@fbertsch review comments have been addressed.

File: schemas/telemetry/mobile-event/mobile-event.1.parquetmr.txt [hunk 5]
```diff
@@ -0,0 +1,45 @@
+message mobile_event {
+  required group metadata {
+    required binary documentId (UTF8);
+    required int64  Timestamp;
+    optional binary Date (UTF8);
+    required binary geoCountry (UTF8);
+    required binary geoCity (UTF8);
+    required binary appBuildId (UTF8);
+    required binary appName (UTF8);
```
Comment (fbertsch) [inline schemas/telemetry/mobile-event/mobile-event.1.parquetmr.txt]: Please add the following fields to `metadata`:

```
required binary appVersion (UTF8);
required double sampleId;
required binary appUpdateChannel (UTF8); 
```

Comment (justindarc) [reply schemas/telemetry/mobile-event/mobile-event.1.parquetmr.txt]: I had to make these `optional` otherwise the tests would fail?

Comment (fbertsch) [reply schemas/telemetry/mobile-event/mobile-event.1.parquetmr.txt]: Looks like you have to ensure the metadata fields are available here: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/tests/hindsight/run/output/test_parquet_schemas.lua#L60

**Comment by justindarc** [comment]
@fbertsch updated to make fields required and updated the tests accordingly. Everything is passing now.

**Comment by justindarc** [comment]
Thanks. I don't have commit access to this repo. Can you hit the squash+merge button for me?

**Pull request merged**